apps/ios/Shared/Views/Chat/ChatItemInfoView.swift

```diff
@@ -275,6 +287,47 @@ struct ChatItemInfoView: View {
         : Color(uiColor: .tertiarySystemGroupedBackground)
     }
 
+    @ViewBuilder private func forwardedFromTab(_ forwardedFromItem: AChatItem) -> some View {
```

Comment:
Rendering full item here via ChatItemView is problematic due to various item interactions. For example, voice message renders as playable but can't be played (probably requires adding player to view).

Accounting for all possible interactions seems pointless as the item copy is readily available - the item for which info was opened.